templates/pool.conf.j2

```diff
@@ -186,10 +187,10 @@ pm.max_requests = {{ pool.pm.max_requests | default(pm_max_requests)}}
 
 ; The log file for slow requests. 
 ; Default value: #INSTALL_PREFIX#/log/php-fpm.log.slow.
-{% if pool.slowlog %}
-    slowlog = {{- pool.slowlog -}}
+{% if pool.slowlog is defined %}
```

Comment:
why are using `is defined` all of the sudden? this use case should have worked before